index.html

```diff
@@ -109,7 +109,7 @@ <h1 class="mb-4">
       <!-- a href="{{ '/about/architecture.html' | url }}" class="btn btn-primary">Learn about WPE Architecture</a -->
     </div>
     <div class="masthead-img" style="padding: 10%;background-color: white"> 
-      <img style="width: 400px;" src="{{ '/assets/wpe-architecture-diagram.png' | url }}"
+      <img style="width: 100%;" src="{{ '/assets/wpe-architecture-diagram.png' | url }}"
```

Comment:
The problem with using `100%` for raster images is that they show blurry if they need to be scaled up whenever what happens to be “100%” is less than the width in pixels of the image. To avoid that we can either:

- Use `width: 400px; max-width: 100%;`, or
- Use `width: 100%;` and replace the PNGs with the SVG (vectorial images can be rendered sharply at any size).

Either way, I think that we will also want to add `text-align: center;` to the parent `<div>` element, so the image will be centered horizontally in the viewport is narrow and the one-column layout is chosen (as is the case for mobile phones).

Reply:
I am very happy to delete this PR and agree with any of the above if you want to submit instead :)  If not, I probably won't get to it for a couple of hours at least.  Or, in the meantime, we could approve this so the site is not broken for mobile and then tweak again asap?